Project3/style.css

```diff
@@ -0,0 +1,152 @@
+figure {
+    position: relative;
+
+
+}
+
+figure img {
+    transition: all 1s;
+    width: 200px;
+    height: auto;
+    margin: 0%;
+}
+
+figure img:hover {
+    transform: scale(1.3);
+}
+
+figcaption {
+    display: none;
+    text-align: left;
+    font-family: 'Heebo', sans-serif;
+    position: relative;
+    width: 50%;
+
+}
+
+h1 {
+    font-family: 'Heebo', sans-serif;
+    text-align: center;
+    color: rgb(255, 255, 255);
+    font-size: 100px;
+    font-display: fallback;
+}
+
+h2 {
+    font-family: 'Mukta Mahee', sans-serif;
+    text-align: left;
+    color: rgb(255, 255, 255);
+    font-size: 50px;
+}
+
+#hour1:target figcaption {
+    display: block;
+    color: rgb(255, 255, 255)
```

Comment:
If you declare this color on `figcaption`, then you wouldn't have to repeat it on every `:target` selector.